### PR TITLE
Bug in class creeation process resolved

### PR DESCRIPTION
**Pull request by kudeepak**
Error displaying at the time of class creation using admin, on line 312. I just make change  if ($data != null && method_exists($data, 'getChildren')) { 

Error was as below : 

Timestamp: Sun Mar 07 2021 09:28:58 GMT+0530 (India Standard Time)
Status: 500 | Internal Server Error
URL: /admin/class/add
Method: POST
Message: method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given
Trace: 
in /var/www/html/vendor/pimcore/pimcore/models/DataObject/ClassDefinition.php:311
#0 /var/www/html/vendor/pimcore/pimcore/models/DataObject/ClassDefinition.php(311): method_exists(NULL, 'getChildren')
#1 /var/www/html/vendor/pimcore/pimcore/models/DataObject/ClassDefinition.php(584): Pimcore\Model\DataObject\ClassDefinition::cleanupForExport(NULL)
#2 /var/www/html/vendor/pimcore/pimcore/models/DataObject/ClassDefinition.php(372): Pimcore\Model\DataObject\ClassDefinition->generateClassFiles(true)
#3 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php(261): Pimcore\Model\DataObject\ClassDefinition->save(true)
#4 /var/www/html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(157): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\ClassController->addAction(Object(Symfony\Component\HttpFoundation\Request))
#5 /var/www/html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(79): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#6 /var/www/html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(195): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#7 /var/www/html/public/index.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#8 {main}

  
  

## Changes in this pull request  
Resolves #

### CODE REVIEW

**Comment by dvesh3** [comment]
not reproducible on our public demo (which is based on 6.8 branch) and master. @kudeepak please mention Pimcore version and steps to reproduce the issue. thanks!

File: bundles/InstallBundle/Command/InstallCommand.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 declare(strict_types=1);
 
 /**
- * Pimcore
+ * Pimcore.
```
Comment (brusch) [inline bundles/InstallBundle/Command/InstallCommand.php:6]:
```suggestion
 * Pimcore
```

File: models/DataObject/ClassDefinition.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Pimcore
+ * Pimcore.
```
Comment (brusch) [inline models/DataObject/ClassDefinition.php:3]:
```suggestion
 * Pimcore
```

**Review by brusch — CHANGES_REQUESTED** [review]
@kudeepak Thanks a lot for your PR! 
Since this is a bug-fix, can you please rebase your PR onto the `6.8` branch? 
Thanks!

**Comment by dvesh3** [comment]
closed due to inactivity.

**Comment by raphdurrer** [comment]
actually same Problem with v6.9.2 on class creation via backend.

**Pull request closed without merging**